tests/TestCase.php

```diff
@@ -10,7 +10,65 @@
 namespace Slim\Tests;
 
 use PHPUnit\Framework\TestCase as PhpUnitTestCase;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+use ReflectionProperty;
+use Slim\Views\Twig;
+use Slim\Views\TwigExtension;
+use Slim\Views\TwigMiddleware;
```

Comment:
This can be removed.

Reply:
Not all, still need ReflectionProperty and Twig

Reply:
Done

Reply:
Sorry, only `use Slim\Views\TwigMiddleware;` should have been removed.
Please re-add `use Slim\Views\TwigExtension;` again.

Reply:
Done